src/components/Message/Message.jsx

```diff
@@ -50,7 +37,7 @@ class Message extends React.Component {
             <div>
                 {this.props.myTurn && box}
                 <div>
-                    <h1>{this.state.message}</h1>
+                    <h1 id="message">{this.props.message}</h1>
```

Comment:
You don't actually need to give this an ID since you can search for an `h1` element in Enzyme. No harm in doing this, just pointing that out.